### PR TITLE
Postgres

### PR DESCRIPTION
**Pull request by vudngo**
@thinkocapo Please review

### CODE REVIEW

File: flask/app.py [hunk 1]
```diff
@@ -76,4 +76,17 @@ def checkout():
     
     process_order(cart)
 
-    return 'Success'
\ No newline at end of file
+    return 'Success'
+
+@app.route('/tool', methods=['GET'])
+def new_tool():
+    with sentry_sdk.start_span(op="db read"):
```
Comment (thinkocapo) [inline flask/app.py]: @vudngo  can you put a try/except block around this? or how are exceptions handled?

File: flask/app.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import os
 from flask import Flask, request, json, abort
 from flask_cors import CORS
-from dotenv import load_dotenv
+from db import add_tool, get_all_tools
```
Comment (thinkocapo) [inline flask/app.py]: keeping the db stuff in its own module is ✅  good

File: flask/db.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+# generate random tool names + descriptions
+def randomString(stringLength=10):
+    """Generate a random string of fixed length"""
+    letters = string.ascii_lowercase
+    return ''.join(random.choice(letters) for i in range(stringLength))
+
+def get_connection():
+    with sentry_sdk.start_span(op="psycopg2.connect"):
+        connection = psycopg2.connect(
+            host = "",
```
Comment (thinkocapo) [inline flask/db.py]: my next PR is for more work on environmental variables for both backend and frontend, so thanks for omitting the values for now.

File: flask/app.py (hunk 1 above)
Comment (thinkocapo) [reply flask/app.py]: can leave it as is for now, can troubleshoot it as we do more complicated things and are throwing exceptions as we develop the demo

Comment (thinkocapo) [reply flask/app.py]: I meant try/except around the `rows = get_all_tools()`

File: flask/db.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    try:
+        cursor.execute(insert_query, (name, tool_type, randomString(10), image, random.randint(10,50)))
+        connection.commit()
+    except:
+        raise "Row insert failed\n"
+    cursor.execute("SELECT * FROM tools ORDER BY ID DESC limit 1")
+    rows = cursor.fetchall()
+    cursor.close()
+    connection.close()
+    return rows
```
Comment (thinkocapo) [inline flask/db.py]: i thought the front-end would just do a full page reload, after adding a item. is the convention I've followed in the past.

Is there a need to fetchall() and return rows at the end of this? can leave it if you want.

Comment (vudngo) [reply flask/db.py]: @thinkocapo the read query was more for me to verify that a new tool was added to the tools table. Let's leave it for now and we can remove it once we get it all working.

Comment (thinkocapo) [reply flask/db.py]: 👍

Comment (thinkocapo) [reply flask/db.py]: @vudngo  if you ever run a script to populate 10,000 rows, maybe remove it at that time.

**Pull request merged**